.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -155,9 +160,41 @@ jobs:
         name: sycl_lit_${{ inputs.build_artifact_suffix }}
         path: lit.tar.xz
 
-  llvm_test_suite:
+  aws-start-matrix:
+    name: Start AWS Matrix
     needs: build
     if: ${{ inputs.lts_matrix != '' }}
+    strategy:
+      fail-fast: false
+      max-parallel: ${{ inputs.max_parallel }}
```

Comment:
We should probably remove max_parallel here to start all 6 aws-start tasks together.

Reply:
Done.